lib/compiler/semantic.js

```diff
@@ -555,6 +578,28 @@ class SemanticPass {
         };
         this.scope = saved_scope;
     }
+    sa_NativeModuleDef(node) {
+        var code = '(function (juttle) {\n' +
+            'var module = {};\n' +
+            node.source + '\n' +
+            'return module.exports; })';
+        var module = eval(code)(runtime);
+        node.source = code;
+        var scope = new Scope(null, 'module');
+        node.elements = [];
```

Comment:
Setting `node.elements` should not be needed as there are no Juttle statements inside a native module.